### DISCUSSION TITLE
How to train items other than person/character

### DISCUSSION THREAD

**Discussion by zhao-binxiang** [bmaltais/kohya_ss#1604] [General]
How do I train a Lora for, say carpet or lamp? 

of course the details need to be correct and accurate.

**Comment by zhao-binxiang** [comment]
any one?

**Comment by madrooky** [comment]
The same way you train a subject. But there are simply to many variables to your general question that I am not willing to go into detail.

**Reply by zhao-binxiang** [reply]
Thanks. But sometimes the patterns aren't exactly the same. For example, I was training a carpet. But the patterns won't be exactly the same.

**Reply by madrooky** [reply]
A carpet is a concept and a pattern is a concept. Maybe you focussed too much on the carpet, which the model knows, but it does not know the pattern. Tried to train the pattern like "pattern xy" and then prompt it like "carpet with pattern xy"? Just an idea. That sort of combination works with all kinds of things otherwise you wouldn't be able to change colours of all sorts of things. Maybe there is a specific form or shape of a carpet you want to have, but that can be totally independent from the pattern on the carpet.

See, that's why i didn't want to get into detail. Just a simple carpet can be incredible complex. :D
